### DISCUSSION TITLE
i have some doubt regarding this repo?

### DISCUSSION THREAD

**Discussion by himanshu9470** [himanshu9470/Hacktoberfest_2024#89] [Q&A] [answered]
can anyone rectify the arrangement of the folders and their files?

**Comment by hari7261** [comment, marked as answer]
Yes we are going to create one new adjustment for your depository and we are really very happy to work with you.

**Comment by himanshu9470** [comment]
alright , then what  are the changes you had been made?

**Reply by hari7261** [reply]
So we are going to arrange all the folders according to the language and according to the weightage of the project also so each and every files of projects will be established as ascending to descending in order.

**Comment by hari7261** [comment]
I would like to say that this idea is great because I think ascending to descending in order is best way for any multiple projects.
